source/includes/k8s/file-transfer-protocol-k8s.rst

```diff
@@ -165,3 +166,73 @@ If SFTP is enabled, the output resembles the following:
 
    enableSFTP: true
 
+.. _minio-certificate-key-file-sftp-k8s
+
+Connect to MinIO Using SFTP with a Certificate Key File
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: RELEASE.2024-05-07T06-41-25Z
+
+
+MinIO supports mutual TLS (mTLS) certificate-based authentication on SFTP, where both the server and the client verify the authenticity of each other.
+
+This type of authentication requires the following public key files:
+
+1. Trusted certificate authority
+2. MinIO Server signed by the certificate authority
+3. User signed by the certificate authority for the client connecting by SFTP and located in the user's ``.ssh`` folder or equivalent for the operating system
```

Comment:
User Certificate?

Reply:
No, you need a public key for the _user_, not a public key for the _user certificate_.
At least, I think that's how the words should work.

Reply:
This as a list where all three items are public key files.
One for for the CA to trust. One for the MinIO Server. One for the user on the client. I've made "public key file" the first set of words for each item in the list to clarify. Since if you missed it, others will too.